### PR TITLE
Refactor @tangleid/did package

### PR DESCRIPTION
**Pull request by ender503**
Since the core functionality of register/resolve identifier was
moved to the @tangleid/core, remove the duplicate function in
@tangleid/did and make the @tangleid/did focus on encoding and decoding
decentralized identifiers.

### CODE REVIEW

File: packages/did/README.md [hunk 1]
```diff
@@ -90,36 +30,47 @@ Used to describe which Tangle network interacts.
...
-    * [~register([options])](#module_did..register)
+    * [~encodeToDid(params)](#module_did..encodeToDid)
+
+    * [~decodeFromDid(tangleid)](#module_did..decodeFromDid)
+
+    * [~createMetaPublicKeys(controller, publicKeyPems)](#module_did..createMetaPublicKeys)
+
 
-    * [~resolver(did, registry)](#module_did..resolver)
+<a name="module_did..encodeToDid"></a>
```
Comment (jserv) [inline packages/did/README.md]: Why?

Comment (ender503) [reply packages/did/README.md]: Could you give me more detail? 🤔

**Comment by jserv** [comment]
Fix the conflict!

**Comment by ender503** [comment]
Rebased.

File: packages/did/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # @tangleid/did
 
-Utilities for Decentralized Identifiers(DIDs) registration and resolve.
+Utilities for decentralized identifiers encoding and decoding.
```
Comment (jserv) [inline packages/did/README.md]: Should be "Utilities for the coding of decentralized identifiers"

File: packages/did/README.template [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # @tangleid/did
 
-Utilities for Decentralized Identifiers(DIDs) registration and resolve.
+Utilities for decentralized identifiers encoding and decoding.
```
Comment (jserv) [inline packages/did/README.template]: Ditto.

File: packages/did/README.md (hunk 2 above)
Comment (ender503) [reply packages/did/README.md]: Fixed.

File: packages/did/README.template (hunk 3 above)
Comment (ender503) [reply packages/did/README.template]: Fixed.

**Pull request merged**